Dockerfile

```diff
@@ -18,7 +18,8 @@ WORKDIR /go/src/sig.k8s.io/gateway-api
 COPY  . .
 ARG TAG
 ARG COMMIT
-RUN CGO_ENABLED=0 GOARCH=amd64 GOOS=linux go build -a -o gateway-api-webhook \
+ARG ARCH
```

Comment:
can replace with https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope